chain/chain/src/garbage_collection.rs

```diff
@@ -1134,133 +1128,104 @@ impl<'a> ChainStoreUpdate<'a> {
     }
 }
 
-/// Returns shards that we tracked in an epoch, given a hash of the last block in the epoch.
-/// The block has to be available, so this function has to be called before gc is run for the block.
+/// Cleans up the state of the parent shard once we stop tracking the epoch where the resharding happened.
```

Comment:
Can you clarify a bit the "tracking the epoch" part?

Reply:
Changed the language a bit